lib/jsonparser.js

```diff
@@ -152,7 +156,8 @@ function formatData(element, descArr) {
             level: currentFormattedDataIndex,
             description: element.description,
             passed: element.passed,
-            sessionId: element.sessionId,
+            //sessionId: element.sessionId,
```

Comment:
not sure.... left it because originally the intent was to have it to differentiate between multiCapabilities running against various resolutions. We could at least clump the tests together. Not needed for now, hence the comment out, but thought it was worth keeping in case we go back to it.